### ISSUE TITLE
Issue with python-publish.yml

### ISSUE THREAD

**Issue by TheShadowOfHassen** [actions/starter-workflows#2063]
I am completely new with GitHub actions. I wanted to add an item to automate releases from my GitHub repository:(https://github.com/TheShadowOfHassen/PygameLord) to my PyPI account. I copied the python-publish.yml to my workflows and added my relevant information.

But whenever I try to make a release, it runs this error:
![image](https://github.com/actions/starter-workflows/assets/94244235/e4942687-3c93-4f7e-bad5-93ecbaf854e2)

I think it's something wrong with the script.

**Comment by JamesMGreene** [comment]
> I think it's something wrong with the script.

Could you elaborate on what specifically you think is unexpected about the script? For example, did you have one or both of those files mentioned as missing in the error text within your repository at the time of that workflow run?

**Comment by TheShadowOfHassen** [comment]
I have a setup.py. I also have the repository here: https://github.com/TheShadowOfHassen/PygameLord If you want to see if there's an issue, that'd be great.

**Comment by TheShadowOfHassen** [comment]
Please don't make this stale. I'm still having the issue. I have no idea why.

Someone's help would be great.

**Comment by TheShadowOfHassen** [comment]
- I'm serious Mr robot I need help and I have no idea what's wrong. Don't mark this as stale. I'm ready to do some work to get this fixed but I need someone to tell you what is wrong

**Comment by TheShadowOfHassen** [comment]
No, Mr robot I still have the problem. I don't know if @JamesMGreene or anyone else is even out there but I still need help.

It's stupid to close issues, the fact that no one talks about them doesn't make them go away.

**Comment by David-Worboys** [comment]
Well I have the same issue and am trying to find a fix as well

**Comment by TheShadowOfHassen** [comment]
Still not fixed. Don’t close
